libs/ui-shared/src/lib/components/molecules/notification/Notification.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 
 //Components
-import { Row, Col } from "imcomponents/atoms/grid";
+import { Row, Col } from "../../atoms/grid";
 
```

Comment:
Can't  we further optimize it to use with atoms path only ?

Reply:
noted